### ISSUE TITLE
Stories using two column template can display duplicate stories in sidebar

### ISSUE THREAD

**Issue by cjg89** [UCF/Today-Child-Theme#86]
**Describe the bug**

Stories using the two column template can display duplicate articles in the sidebar, under "More Headlines", "More about [Primary Category]", and/or "More About [Primary Tag]", if those stories are published recently and/or contain similar primary categories/tags to the current story.

Ideally, stories in the sidebar should not repeat, to not only reduce repetition + encourage further reading, but to also resolve issues with ARIA landmarks having identical names on the page.

**Steps to Reproduce**

See screenshot below.  Occurs on stories using the two column template that have similar categories and tags to the current story, and/or are published recently.

**Expected Behavior**

All sidebar stories should be unique.

**Possible Fix**

1. Update `today_get_post_cat_headlines()` to exclude stories returned by `today_get_post_more_headlines()`.
2. Update `today_get_post_tag_headlines()` to exclude stories returned by `today_get_post_more_headlines()` AND `today_get_post_cat_headlines()`.

**Your environment:**

n/a

**Additional context**

Example:
![Screen Shot 2019-08-26 at 10 14 31 AM](https://user-images.githubusercontent.com/1649216/63697166-58f16300-c7ea-11e9-874b-395ab9fef676.png)
